### PR TITLE
Restructure Home Page

### PR DESCRIPTION
**Pull request by medhaanand**

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -106,39 +110,31 @@ module.exports = () => (
...
-        {
-          title: 'Facebook',
-          url: '/other/facebook'
-        },
-        {
-          title: 'Native support',
-          url: 'https://hellohaptik.github.io/javascript_sdk/native-support'
+          title: 'External Agent Chat Tool',
+          url: '/external-chat-tool'
         }
```
Comment (nikunj1705) [inline website/pages/en/index.js:138]: The SDKs will go in a different section? Also, ECT should get more prominence, no?

File: website/pages/en/index.js [hunk 2]
```diff
@@ -54,14 +54,14 @@ module.exports = () => (
         {
           title: 'Bot Builder',
           description:
-            'Everything required to get you from beginner to advanced in Bot Building & Development',
-          ctaText: 'Learn More',
+            'Everything required to get you from beginner to advanced in Bot Building',
```
Comment (tejas-haptik) [inline website/pages/en/index.js:57]: No full stops? Across the descriptions?

Comment (medhaanand) [reply website/pages/en/index.js:57]: No fullstop, it is currently also not there and that looks good so haven't changed that

File: website/pages/en/index.js (hunk 1 above)
Comment (medhaanand) [reply website/pages/en/index.js:138]: As ECT is a very different way of integrations as compared to other integrations in this category. The idea is we create a new section called External Integrations where we have WA, FB, Bot API, ECT.  but till that is done, I wanted to put it under **Other** as it is different than others

**Pull request merged**